openquake/hazardlib/tom.py

```diff
@@ -220,3 +220,150 @@ def get_pnes(rate, probs, poes, time_span):
         for p, prob in enumerate(probs[1:], 1):
             pnes[:] += prob * (1 - poes) ** p
         return pnes.clip(0., 1.)  # avoid numeric issues
+
+
+class NegativeBinomialTOM(BaseTOM):
+    """
+    Negative Binomial temporal occurrence model.
+    """
+    def __init__(self, time_span, occurrence_rate=None, parameters=None):
```

Comment:
I don't like to use a generic `parameters`. Just use mu and alpha, as mandatory parameters. Also the occurrence_rate is not needed, see https://github.com/gem/oq-engine/pull/7931